pkg/networkmanager/network-interface-members.jsx

```diff
@@ -83,10 +80,7 @@ export const NetworkInterfaceMembers = ({
                         } else if (dev) {
                             with_checkpoint(
                                 model,
-                                function () {
-                                    return dev.disconnect()
-                                            .fail(show_unexpected_error);
-                                },
+                                () => dev.disconnect().catch(show_unexpected_error),
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19120-20230722-055749-01048432-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)